iroha/src/sumeragi.rs

```diff
@@ -45,8 +51,65 @@ impl Message {
     }
 }
 
+/// Structure describing a request to other peers to change view because of the commit timeout.
```

Comment:
Maybe "`CommitTimeout` message structure used to notify peers about ..."? And what view?